docs/parser.md

```diff
@@ -172,43 +182,57 @@ require("@babel/parser").parse("code", {
 | `typescript` ([repo](https://github.com/Microsoft/TypeScript)) | `var a: string = "";` |
 | `v8intrinsic` | `%DebugPrint(foo);` |
 
-#### ECMAScript [proposals](https://github.com/babel/proposals)
+#### ECMAScript [proposals](https://github.com/tc39/proposals)
```

Comment:
I'm okay with leaving this pointed to our proposals repo as it has additional Babel-related info.

@hzoo thoughts?

Reply:
That's fine, but we haven't been keeping as much up to date as we could to bring more of that context to light (though it shouldn't be as hard to do so..)

Reply:
```suggestion
#### ECMAScript [proposals](https://github.com/babel/proposals)
```
Reverted this change, we can discuss how to deal with our proposals repo later.